app/_posts/2024-04-17-kuma-2-7-0.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Kuma 2.7 release with GUI improvements, shadow policies and more...
+description: Kuma 2.7 release with GUI improvements, shadow policies and more...
+date: 2024-04-17
+headerImage: /assets/images/blog/test-header.jpg
+tags:
+  - Release
+---
+
+We’re excited to announce the release of Kuma 2.7, as usual the release is packed with new features and improvements.
+
+We've picked these highlights for you:
+
+- GUI improvements for easier management of your service mesh
+- GatewayAPI integration is now Generally Available and uses MeshHTTPRoute
+- Shadow policies to help you migrate to the new policies
+- Better `MeshMetric` OpenTelemetry support and profiles
+
+You can always check our [release notes](https://github.com/kumahq/kuma/releases/tag/2.7.0) for the full list of changes.
+
+
+## GUI improvements
+
+List views now displays names, namespaces, and zones for policies and dataplanes, providing a clear understanding of resource placement in multi-zone environments:
+
+![List of kuma dataplanes showing the namespace](/assets/images/blog/kuma_2_7_dpp_list.png)
+
+The GUI now offers a dedicated view of routes managed by your built-in gateway, providing a clear overview of your gateway configuration:
+
+![A kuma gateway showing the routes in use](/assets/images/blog/kuma_2_7_gateway_view.png)
+
+The dataplane view now displays policies applied to inbound and outbound traffic, simplifying proxy behavior comprehension:
+
+![A kuma gateway showing the routes in use](/assets/images/blog/kuma_2_7_dpp_policies.png)
+
+## GatewayAPI integration
+Our Gateway API integration now uses [`MeshHTTPRoute`](/docs/2.7.x/policies/meshhttproute), enabling us to retire `MeshGatewayRoute` which will be deprecated and removed in the future.
+We've also documented how to [use the Gateway API with multi-zone](/docs/2.7.x/using-mesh/managing-ingress-traffic/gateway-api/#multi-zone-deployments).
+
+Additionally, we're thrilled to announce that our entire [Gateway API](https://gateway-api.sigs.k8s.io/) integration, including GAMMA support, is now Generally Available and no longer considered experimental.
+
+## Shadow policies and migration of policies
+
+Since Kuma 2.0 we're rolling out targetRef policies. These policies are more powerful and easier to use than the previous version.
```

Comment:
```suggestion
Since Kuma 2.0 we've been rolling out targetRef policies. These policies are more powerful and easier to use than the previous version.
```

Not totally sure this what you mean here put pretty sure this is what it should be?